javapackages-tools.spec

```diff
@@ -134,31 +134,15 @@ This package provides non-essential macros and scripts to support Java packaging
 %setup -q -n %{pkg_name}-%{version}
 
 %build
-%configure --pyinterpreter=%{python_interpreter} %{?scl:--rpmconfigdir=%{_root_prefix}/lib/rpm --scl=%{scl} --scl_root=%{_scl_root}}
+%configure --pyinterpreter=%{python_interpreter} %{?scl:--rpmconfigdir=%{_root_prefix}/lib/rpm --root_sysconfdir=%{_root_sysconfdir} --scl=%{scl} --scl_root=%{_scl_root}}
```

Comment:
I think it would be better to pass `%rpmmacrodir` instead, with default value of /etc/rpm/macros.d, and install macro files there, regardless of whether SCL built or not.